diff --git a/ffffff/_vendor/black.py b/ffffff/_vendor/black.py
index 651e600..0bead1c 100644
--- a/ffffff/_vendor/black.py
+++ b/ffffff/_vendor/black.py
@@ -38,6 +38,8 @@ from typing import (
     cast,
 )
 
+import ffffff
+
 from ffffff._vendor.appdirs import user_cache_dir
 from ffffff._vendor.attr import dataclass, evolve, Factory
 from ffffff._vendor import click
@@ -54,7 +56,7 @@ from ffffff._vendor.blib2to3.pgen2.parse import ParseError
 
 from ffffff._vendor._black_version import version as __version__
 
-DEFAULT_LINE_LENGTH = 88
+DEFAULT_LINE_LENGTH = 79
 DEFAULT_EXCLUDES = r"/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|\.svn|_build|buck-out|build|dist)/"  # noqa: B950
 DEFAULT_INCLUDES = r"\.pyi?$"
 CACHE_DIR = Path(user_cache_dir("black", version=__version__))
@@ -354,7 +356,7 @@ def read_pyproject_toml(
         "ignored due to --exclude=."
     ),
 )
-@click.version_option(version=__version__)
+@click.version_option(version=__version__ + '+' + ffffff.__version__)
 @click.argument(
     "src",
     nargs=-1,
@@ -2886,7 +2888,7 @@ def normalize_string_quotes(leaf: Leaf) -> None:
     if new_escape_count > orig_escape_count:
         return  # Do not introduce more escaping
 
-    if new_escape_count == orig_escape_count and orig_quote == '"':
+    if new_escape_count == orig_escape_count and orig_quote == "'":
         return  # Prefer double quotes
 
     leaf.value = f"{prefix}{new_quote}{new_body}{new_quote}"
